src/Common/Validation/EmailAddressValidator.cs

```diff
@@ -0,0 +1,51 @@
+using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace Passwordless.Common.Validation;
+
+public static class EmailAddressValidator
```

Comment:
We could just use `MailAddress` from `System.Net.Mail` instead of maintaining our own validation for email addresses.